### ISSUE TITLE
pytorch training

### ISSUE THREAD

**Issue by Hackerlil** [ialhashim/DenseDepth#40]
when I try to training pytorch with all your setting in origin python implementations,it seems loss
decrease when begin but wave at val loss about 0.19x_ after about 3-4 epoch training is it normal?
or the pytorch implemention depends greatly on initialize? thank you

**Comment by ialhashim** [comment]
There are differences between the PyTorch and the Tensorflow (Keras) models. 

The performance, especially at the first couple of epochs is highly dependent on the encoder's orginal performance in classification. The pre-trained one provided by Keras is better than the one in PyTorch (I think I am also using a slightly smaller DenseNet densenet161). 

Also the loss function is not an exact match. If you figure out a better way to make the PyTorch performance better please submit a pull request, I would greatly appreciate it.

One thing to also play with is the learning rate. Start with something like 0.0002 and then step down two or three times until doing one epoch of 0.00001. There is no magic formula, I usually find the best learning rate schedule experimentally.
